tests/TextUI/test-order-randomized-with-dependency-resolution.phpt

```diff
@@ -18,7 +18,7 @@ PHPUnit %s by Sebastian Bergmann and contributors.
 Runtime:       %s
 Random seed:   %d
 
-.....                                                               5 / 5 (100%)
+.....
```

Comment:
:-1: for dropping info how far we are with execution

Reply:
a percentage of an unknown number of testcases it impossible to calculate, so he options were having over 100% tests executed or just showing the number of tests executed.
The only reason there is no number in this specific line is, that it's the last one and directly beneath it the result - including the number of lines - is shown (multiple lines, see: https://github.com/sebastianbergmann/phpunit/pull/3210/files/47fdd0b8a6309e1534190ef8b76a97fff0d919ac#diff-cc5dd71ca2e247cdaa4cde8688137e59R18 ).

I picked the later one for now since I didn't see a way of doing the counting properly, see https://github.com/sebastianbergmann/phpunit/issues/10#issuecomment-404713231

Reply:
When additional tests are discovered the total number of tests can be increased making the percentage work again.

Reply:
will also lead to very weird cases, let's take the following:
Suite1: 100 Tests, 20 DataProviderTestsuites -> initial estimation 120 tests
Assuming that doesn't fit into a line we could end up with
```
... 80/120 (66%)
... 160/160 (100%)
... 180/180 (100%)
```
I think this is even more confusing.

Reply:
Just wondering, would keeping the count itself work for you? That would bypass the issue of having the target number change but at least provide some information on where we are at? @keradus ?